reana_cluster/backends/kubernetes/templates/clusterrole/service-reader.yaml

```diff
@@ -2,7 +2,7 @@
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  namespace: default
+  # namespace: {{NAMESPACE}}
```

Comment:
This is something we had incorrectly set, there is no namespace for `clusterroles`, since they are cluster-wide, good catch!